src/Menu/SideDrawer.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component, Fragment } from 'react';
+import { FaRegEye, FaDatabase } from "react-icons/fa";
+import { IoIosLogOut, IoIosSettings, } from "react-icons/io";
+import { GoChevronLeft } from "react-icons/go";
+import { MdTagFaces } from "react-icons/md";
+import AssignmentIcon from '@material-ui/icons/Assignment';
+import classes from "./SideDrawer.css"
+import SideMenus from "../Menu/SideMenus.js";
+
+
+class SideDrawer extends Component {
+  constructor() {
+    super();
+    this.state = {
+      admin: [
+        { name: 'Configure Subcourse', icon: IoIosSettings },
+        { name: 'View Evaluation', icon: FaRegEye },
+        { name: 'Assign Evaluator', icon: MdTagFaces },
+        { name: 'Logout', icon: IoIosLogOut }
+      ],
+      trainee: [
+        { name: 'View Evaluation', icon: FaRegEye },
+        { name: 'Logout', icon: IoIosLogOut }
+      ],
+      evaluator: [
+        { name: 'Evaluate', icon: AssignmentIcon },
+        { name: 'Logout', icon: IoIosLogOut }
+      ],
+
+      showmenu: false,
+
+      user : 'admin',
+
+    };
+  }
+
+  showMenu = () => {
+    this.setState({ showmenu: !this.state.showmenu }
+    );
```

Comment:
why seperate line for this?

Reply:
This is a function to close and open the collapsive menus, ok i will change that in to a single line code

Reply:
Not changed yet

Reply:
This is a function, i changed that into a single line